src/components/Footer/__snapshots__/index.test.js.snap

```diff
@@ -1,69 +1,57 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
zullen we dan overal de snapshots weghalen?

Reply:
Slowly but surely replace them, yes.